include/oneapi/dpl/pstl/onedpl_config.h

```diff
@@ -117,7 +117,8 @@
 
 // Enable loop unrolling pragmas where supported
 #if (__INTEL_LLVM_COMPILER || __INTEL_COMPILER ||                                                                      \
-     (!defined(__INTEL_LLVM_COMPILER) && !defined(__INTEL_COMPILER) && _ONEDPL_GCC_VERSION >= 80000))
+     (!defined(__INTEL_LLVM_COMPILER) && !defined(__INTEL_COMPILER) &&                                                 \
+      ((_ONEDPL_GCC_VERSION >= 80000) || (_ONEDPL_CLANG_VERSION >= 40000))))
```

Comment:
Maybe we can use the version `37000`? I can see in the [Clang 3.7 docs](https://releases.llvm.org/3.7.0/tools/clang/docs/AttributeReference.html#pragma-unroll-pragma-nounroll) that `#pragma unroll` is valid whereas it is absent in the [Clang 3.6 docs](https://releases.llvm.org/3.6.0/tools/clang/docs/AttributeReference.html).

Reply:
Good point! Done.